langtest/augmentation/__init__.py

```diff
@@ -350,7 +352,7 @@ def fix(
         """
         df = DataFactory(training_data, self.__task)
         data = df.load()
-        new_data = []
+        new_data = data.copy() if append_original else []
```

Comment:
it only works when data is in `pd.Dataframe` or `pd.Series`. for samples, it doesn't take a copy of each sample.